src/components/BookForm.js

```diff
@@ -17,21 +17,23 @@ function BookForm() {
 
   return (
     <div className="add-book-form">
-      <h3>ADD NEW BOOK</h3>
+      <h3 className="add-new-book">ADD NEW BOOK</h3>
       <form>
         <input
+          className="new-book-title"
           value={title}
           type="text"
           placeholder="Book Title"
           onChange={(e) => { setTitle(e.target.value); }}
         />
         <input
+          className="new-book-author"
           value={author}
           type="text"
           placeholder="Author"
           onChange={(e) => { setAuthor(e.target.value); }}
         />
-        <button onClick={handleClick} type="submit">Add Book</button>
+        <button className="add-btn" onClick={handleClick} type="submit">ADD BOOK</button>
       </form>
     </div>
```

Comment:
- [ ] Please stick to the [Zaplin](https://zeplin.io/) design but your design is not the same as [Zaplin](https://zeplin.io/) design so kindly fix all design issues :+1: check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-06-24 23-29-07](https://github.com/sunga12/bookstore/assets/45226848/6117aa92-e06a-44e6-b90b-d6682461210c)

**Zaplin Design**
![Screenshot from 2023-06-24 23-27-20](https://github.com/sunga12/bookstore/assets/45226848/cabe40e2-7865-404e-8b74-d012a9178941)

- [ ] If you check your design the width and height of your `input fields` and `button` is not the same as [Zaplin](https://zeplin.io/) design so kindly reduce the height and width of `input fields` and `button` :+1: 
- [ ] If you check the spacing between the `ADD NEW BOOK` text the `input field` is not the same as [Zaplin](https://zeplin.io/) design so kindly fix this issue as well :+1: